cpp/benchmarks/join/join_common.hpp

```diff
@@ -53,10 +53,28 @@ struct null75_generator {
   }
 };
 
+enum join_t { CONDITIONAL, MIXED, HASH };
```

Comment:
nit: since we are using scoped enums anyway at all places, this could be `enum class`.

Reply:
Sure, but the file has no scoped namespace so I don't think it matters. I will add it anyway